lib/array.js

```diff
@@ -147,6 +147,15 @@ internals.checkItems = function (items, wasArray, state, options) {
             continue;
         }
 
+        // Ordered
+        if (this._flags.ordered) {
```

Comment:
This part is missing re-assignment of the value if it's valid, the abortEarly case and the strip option (see further parts for how to).
If think error or not it should always continue, I don't see any reason not to.